webrender/src/render_backend.rs

```diff
@@ -24,6 +24,9 @@ use tiling::FrameBuilderConfig;
 use gleam::gl;
 use offscreen_gl_context::GLContextDispatcher;
 
+/// The render backend is responsible for transforming high level display lists into
+/// GPU-friendly work which is then submitted to thre renderer.
```

Comment:
might be worth mentioning that it produces frames (`Frame`), which are sent to the renderer